packages/mermaid/src/docs/community/development.md

```diff
@@ -1,372 +1,81 @@
 # Contributing to Mermaid
 
-## Contents
-
-- [Technical Requirements and Setup](#technical-requirements-and-setup)
-- [Contributing Code](#contributing-code)
-- [Contributing Documentation](#contributing-documentation)
-- [Questions or Suggestions?](#questions-or-suggestions)
-- [Last Words](#last-words)
-
----
+> You can pick [Native](/community/development) or [Docker](/community/docker-development) installation guide.
 
 So you want to help? That's great!
 
 ![Image of happy people jumping with excitement](https://media.giphy.com/media/BlVnrxJgTGsUw/giphy.gif)
 
 Here are a few things to get you started on the right path.
 
-## Technical Requirements and Setup
+## Get the Source Code
 
-### Technical Requirements
+In GitHub, you first **fork** a repository when you are going to make changes and submit pull requests.
 
-These are the tools we use for working with the code and documentation.
-
-- [volta](https://volta.sh/) to manage node versions.
-- [Node.js](https://nodejs.org/en/). `volta install node`
-- [pnpm](https://pnpm.io/) package manager. `volta install pnpm`
-- [npx](https://docs.npmjs.com/cli/v8/commands/npx) the packaged executor in npm. This is needed [to install pnpm.](#2-install-pnpm)
+Then you **clone** a copy to your local development machine (e.g. where you code) to make a copy with all the files to work with.
 
-Follow [the setup steps below](#setup) to install them and verify they are working
+[Fork mermaid](https://github.com/mermaid-js/mermaid/fork) to start contributing to the main project and its documentaion, or [search for other repositories](https://github.com/orgs/mermaid-js/repositories).
 
-### Setup
+[Here is a GitHub document that gives an overview of the process.](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
 
-Follow these steps to set up the environment you need to work on code and/or documentation.
+## Technical Requirements
 
-#### 1. Fork and clone the repository
+> You can pick [Native](/community/development) or [Docker](/community/docker-development) installation guide.
```

Comment:
```suggestion
> The following documentation describes how to work with Mermaid in your host
> environment. There's also a
> [Docker installation guide](../community/docker-development.md) if you prefer
> to work in a Docker environment.
```

Two things:
  - The grammar for this bit is a bit confusing.
  - We should use relative links (e.g. `../xxxx.md` instead of `/xxxx`) when possible.

Reply:
Fixed